pkg/sessions/redis/redis_store.go

```diff
@@ -108,6 +116,20 @@ func newRedisCmdable(opts options.RedisStoreOptions) (Client, error) {
 	return newClient(client), nil
 }
 
+// parseRedisURLs parses a list of redis urls and returns a list
+// of addresses in the form of host:port that can be used to connnect to Redis
+func parseRedisURLs(urls []string) ([]string, error) {
+	addrs := []string{}
+	for _, u := range urls {
+		parsedURL, err := redis.ParseURL(u)
+		if err != nil {
+			return nil, fmt.Errorf("unable to parse redis url: %s", err)
```

Comment:
It's idiomatic to use %v for errors 🙂 
```suggestion
			return nil, fmt.Errorf("unable to parse redis url: %v", err)
```